Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectControlled.mo

```diff
@@ -0,0 +1,299 @@
+  Buildings.Controls.Continuous.LimPID con(
+    final controllerType=controllerType,
+    final k=k,
+    final Ti=Ti,
+    final Td=Td,
+    final yMax=yMax,
+    final yMin=yMin,
+    reverseActing=false,
+    y_reset=0)
+    "District return temperature controller"
+    annotation (Placement(transformation(extent={{-220,10},{-200,-10}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiSup(
+    redeclare final package Medium = MediumSer,
+    final m_flow_nominal=mBui_flow_nominal)
+    "Building supply temperature sensor"
+    annotation (Placement(transformation(extent={{230,190},{250,210}})));
+  Modelica.Blocks.Logical.OnOffController onOffCon(bandwidth=0.2)
+    "On off control for the control valve"
+    annotation (Placement(transformation(extent={{-180,60},{-160,80}})));
+  Modelica.Blocks.Logical.Switch swi
+    "Switch between full opening and PI control signal."
+    annotation (Placement(transformation(extent={{-80,60},{-60,80}})));
+  Modelica.Blocks.Sources.Constant const(k=1)
+    "Full opening of the control valve."
+    annotation (Placement(transformation(extent={{-160,120},{-140,140}})));
+  Modelica.Blocks.Logical.Not notCon
+    "Reverse the on/off signal"
+    annotation (Placement(transformation(extent={{-140,60},{-120,80}})));
+protected
+  final parameter MediumSer.ThermodynamicState sta_default=MediumSer.setState_pTX(
+    T=MediumSer.T_default,
+    p=MediumSer.p_default,
+    X=MediumSer.X_default)
+    "Medium state at default properties";
+  final parameter Modelica.Units.SI.SpecificHeatCapacity cp_default=MediumSer.specificHeatCapacityCp(
+    sta_default)
+    "Specific heat capacity of the fluid";
+equation
+  connect(port_aSerCoo, senTDisSup.port_a)
+    annotation (Line(points={{-300,-280},{-180,-280}}, color={0,127,255}));
+  connect(senTDisSup.port_b, senMasFlo.port_a)
+    annotation (Line(points={{-160,-280},{-120,-280}}, color={0,127,255}));
+  connect(senMasFlo.port_b, jun.port_1)
+    annotation (Line(points={{-100,-280},{-60,-280}}, color={0,127,255}));
+  connect(ports_aChiWat[1], senTBuiRet.port_a)
+    annotation (Line(points={{-300,200},{-220,200}}, color={0,127,255}));
+  connect(senTBuiRet.port_b, spl.port_1)
+    annotation (Line(points={{-200,200},{-60,200}}, color={0,127,255}));
+  connect(spl.port_2, conVal.port_a)
+    annotation (Line(points={{-40,200},{-10,200}}, color={0,127,255}));
+  connect(senMasFlo.m_flow, pro.u2)
+    annotation (Line(points={{-110,-269},{-110,-192},
+          {90,-192},{90,-116},{118,-116}}, color={0,0,127}));
+  connect(pro.y, cp.u)
+    annotation (Line(points={{141,-110},{180,-110}}, color={0,0,127}));
+  connect(cp.y, Q_flow)
+    annotation (Line(points={{203,-110},{320,-110}}, color={0,0,127}));
+  connect(cp.y, int.u)
+    annotation (Line(points={{203,-110},{204,-110},{204,-150},
+          {258,-150}}, color={0,0,127}));
+  connect(int.y, Q)
+    annotation (Line(points={{281,-150},{320,-150}}, color={0,0,127}));
```

Comment:
For this connection, we cannot see the `Q`.

Reply:
`Q` is visualized by adding graphic annotation.